convex/schema.ts

```diff
@@ -0,0 +1,26 @@
+import { defineSchema, defineTable } from "convex/server";
+import { v } from "convex/values";
+
+// Why is there a different schema for game and room??? -  Jakob
```

Comment:
Yes, I think we can merge the schemas

Reply:
It's the office code, And I use the Convex at the first time, so I don't know may be you are right, but I don't know, sorry